docs/github-action.md

```diff
@@ -53,7 +53,7 @@ suggestions based on that local diff](https://github.com/getsentry/action-git-di
 
 | Name | Description | Required | Default |
 |------|-------------|----------|---------|
-| files| Files or patterns to check | false | If not defined, entire repository is checked |
+| files| Files or patterns to check | false | If not defined, entire repository is checked except hidden files |
```

Comment:
Except I believe the config would be respected for any excludes.

Let's instead say "If not defined, the default set of files are checked"

Reply:
Done 🍏

Reply:
I just badly need git index-based checking instead of filesystem-based checking.